apps/tasks/views.py

```diff
@@ -0,0 +1,36 @@
+from rest_framework import status, filters
+from rest_framework.decorators import action
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.viewsets import ModelViewSet
+
+from apps.tasks.models import Task
+from apps.tasks.serializers import TaskSerializer, TaskAssignSerializer, DisplayTaskSerializer
+
+
+class TaskViewSet(ModelViewSet):
+    serializer_class = TaskSerializer
+    queryset = Task.objects.all()
+    permission_classes = (IsAuthenticated,)
+    filter_backends = [filters.SearchFilter]
+    search_fields = ['=id']
+
+    def perform_create(self, serializer):
+        serializer.save(user=self.request.user)
+
+    @action(detail=False, serializer_class=DisplayTaskSerializer)
+    def display_my(self, request, *args, **kwargs):
```

Comment:
Acum endpointul arata asa: `/tasks/display_my`, dupa good practice e bine ca endpointul sa nu contina `_`. Poti sa adaugi parametru `url_path` ca sa specifici alt nume, ar arata bine `/tasks/my`, adica `url_path="my"`.

Uite un exemplu din documentatie cu explicatie mai larga:
![image](https://github.com/korsarking/ebs-python-internship-test/assets/53467630/7d6af1f5-e3ae-45bd-bea7-24679f892f2f)
https://www.django-rest-framework.org/api-guide/routers/

Reply:
Deleted `display_my` because we can use `DjangoFilterBackend` with `http://127.0.0.1:8000/tasks?status=Completed&user_id=3`